lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -213,6 +213,13 @@ class LighthouseReportViewer {
 
       const features = new ViewerUIFeatures(dom, saveCallback);
       features.initFeatures(json);
+
+      // No UI for treemap yet. For now, must run this command in console.
+      // @ts-ignore
+      globalThis._tmpFeatures = features;
+      const command = '_tmpFeatures._openTreemap();';
```

Comment:
Until there is a way to open the treemap viewer in the report, I wanted a way to do in simply through the console.